Packs/CommonPlaybooks/Playbooks/playbook-Dedup_-_Generic_v4.yml

```diff
@@ -166,15 +186,16 @@ tasks:
     quietmode: 0
     isoversize: false
     isautoswitchedtoquietmode: false
+    continueonerrortype: ""
   "11":
     id: "11"
-    taskid: a58c04dc-bed0-47ed-8256-cfd360343866
+    taskid: b987c1f7-ae48-442b-835c-99b7d380cdb2
     type: regular
     task:
-      id: a58c04dc-bed0-47ed-8256-cfd360343866
+      id: b987c1f7-ae48-442b-835c-99b7d380cdb2
       version: -1
       name: Link similar incident
-      description: commands.local.cmd.linkIncidents
+      description: Link incidents together or mark them as duplicate and close them.
```

Comment:
Was that changed by the formater or by you? if it's you, please leave it as it was.

Reply:
changed it back to the 'commands.local.cmd.linkIncidents' description.
Why this description was set for the task?